plugins/package-managers/swiftpm/src/main/kotlin/SwiftPm.kt

```diff
@@ -17,6 +17,8 @@
  * License-Filename: LICENSE
  */
 
+@file:Suppress("TooManyFunctions")
```

Comment:
Nit: Maybe start to move out extension functions to a separate file in a follow-up commit?